src/app/shared/components/footer/footer.component.ts

```diff
@@ -37,4 +45,27 @@ export class FooterComponent implements OnInit {
     window.scrollTo(0, 0);
   }
 
+  getCurrentUser() {
+    this.userService.userLoaded$.subscribe((data) => {
+      if (data) {
+        this.currentUser = this.userService.currentUser;
+      } else {
+        this.currentUser = null;
+      }
+    });
+  }
+
+  submitConsultation() {
+    if (!this.currentUser) {
+      this.router.navigateByUrl('/auth');
+      return;
+    } else {
+      if (this.currentUser && this.currentUser.confirmedAt) {
```

Comment:
```suggestion
      if (this?.currentUser?.confirmedAt) {
```

Not sure if it is supported though. Please test

Reply:
Optional chaining is supported from node v14. For civis we are using v10.15.3.
To check I changed node version to 10.15.3 using nvm, but optional chaining was still working. 
I have changed the code for now to check build process on staging.